src/Chapter1.hs

```diff
@@ -556,7 +561,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+  | c == 'a' || c == 'e' || c == 'o' || c == 'i' || c == 'u' = True
+  | otherwise = False
```

Comment:
Since both `==` and `||` operators return `Bool`, you can avoid using guards and just return the result of the or-expression:

```suggestion
isVowel c = c == 'a' || c == 'e' || c == 'o' || c == 'i' || c == 'u'
```